certified-connectors/tyntec-Viber/apiDefinition.swagger.json

```diff
@@ -20,32 +20,33 @@
   "paths": {
     "/chat-api/v2/messages/": {
       "post": {
-        "responses": {
-          "default": {
-            "description": "default",
-            "schema": {
-              "type": "object",
-              "properties": {
-                "messageId": {
-                  "type": "string",
-                  "description": "messageId"
-                }
+        "202": {
```

Comment:
@ridlees You have removed "responses" property also which is a syntax error.

Reply:
fixed in https://github.com/microsoft/PowerPlatformConnectors/pull/438/commits/b5d8c7da3c5c771af90dd9666acbb25fd92b43fa